packages/desktop-client/src/components/FinancesApp.js

```diff
@@ -149,6 +161,24 @@ function StackedRoutesInner({ location }) {
         element={isNarrowWidth ? <MobileAccount /> : <Account />}
       />
 
+      <Route
+        path="/accounts/:id/:transactionId"
```

Comment:
What do you think of `/accounts/:id/transaction/:transactionId`? 
Then the new txn route below might be: `/account/:id/transaction/new`.

Reply:
I like that. I think `/accounts/:id/transactions/:transactionId` would be best so that `/accounts` / `/transactions` are both plural 🤔

Reply:
Updated the routes in https://github.com/actualbudget/actual/pull/1340/commits/3bc55d6ce9774b6bbd5050d6aa2ca8cf8fb484b2